plugins/qeta-backend/src/service/router.ts

```diff
@@ -744,6 +742,52 @@ export async function createRouter({
     response.send(tags);
   });
 
+  // POST /attachments
+  router.post(
+    '/attachments',
+    multerUploaderMiddleware(config, 'image'),
+    async (request, response) => {
+      const backendBaseUrl = config.getString('backend.baseUrl');
+      const qetaUrl = `${backendBaseUrl}/api/qeta/attachments`;
+
+      const storageType =
+        config.getOptionalString('qeta.storage.type') || 'database';
+
+      const imageBuffer = await fs.promises.readFile(`${request.file?.path}`);
```

Comment:
I don't really follow what is this reading? The file is already included in the request so what is read from the filesystem?